smoketest.sh

```diff
@@ -180,7 +180,8 @@ function createPod() {
         --publish 9990:9990 \
         --publish 9991:9991 \
         --publish 10000:10000 \
-        --publish 10001:10001
+        --publish 10001:10001 \
+        --publish 10010:10010
```

Comment:
Small thing: add an extra comment on about what the port 10010 is for (like the others)?